prep-css-box-model/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>CSS Box Model</title>
+  <link rel="stylesheet" href="style.css">
+  <style>
+    button{
+      margin: 20px;
+      padding: 10px;
+      border-style: groove double, color: darkgray;
```

Comment:
Hi Laura, your border styling isn't applying because there's a comma here instead of a semicolon after `double`.